ebi_eva_common_pyutils/config_utils.py

```diff
@@ -14,6 +14,9 @@
 
 from urllib.parse import quote_plus
 from lxml import etree as et
+import json
+import yaml
+import urllib.request
```

Comment:
Hoping none of the above needs to be part of setup.py pre-requisite libraries...

Reply:
I don't think they will, only `retry` that was recently added and included in the setup

Reply:
`yaml` needs to have `pyyaml` added to the prerequisite

Reply:
That has been addressed.